docs/references/beamfits_memo.tex

```diff
@@ -0,0 +1,160 @@
+\textit{beamfits} files which have small differences in format. There are also several optional 
+metadata components that may or may not be present in any given file. These variations are 
+described in detail in the following sections.
+
+\section{Primary HDU}
+The primary HDU of \textit{beamfits} files is an Image HDU containing the beam model.
+
+\subsection{Primary Header}
+The following are required keywords in the primary header of a  \textit{beamfits} file. For more detailed explanations 
+of what these keywords mean, see the descriptions on pyuvdata's ReadTheDocs uvbeam\_parameters page. 
+The UVBeam parameter corresponding to each keyword is noted in parentheses.
+As with all FITS files, HISTORY and COMMENT cards are allowed.
+
+\begin{itemize}
+\item{\textbf{BTYPE}: \textit{string} should be set to either `power' or `efield' to designate a power or E-field beam type. (beam\_type)}
+\item{\textbf{NORMSTD}: \textit{string} should be set to one of `physical', `peak' or `solid\_angle'. (data\_normalization)}
+\item{\textbf{COORDSYS}: \textit{string} should be set to one of `az\_za' (for an azimuth, zenith angle regular grid), `orthoslant\_zenith' (for an orthoslant projection at zenith) or `healpix'. (pixel\_coordinate\_system)}
+\item{\textbf{TELESCOP}: \textit{string} Telescope name. (telescope\_name)}
+\item{\textbf{FEED}: \textit{string} Feed name (used to distinguish between different physical feed types on the same telescope). (feed\_name)}
+\item{\textbf{FEEDVER}: \textit{string} Feed version (used to distinguish between different versions of the same feed on the same telescope). (feed\_version)}
+\item{\textbf{MODEL}: \textit{string} Model name  (used to distinguish between different models of the same hardware). (model\_name)}
+\item{\textbf{MODELVER}: \textit{string} Model version (used to distinguish between different versions of the same models of the same hardware). (model\_version)}
+\item{\textbf{FEEDLIST}: \textit{string list} Only for E-field beams. The list of feeds orientations represented in the data. (feed\_array)}
+\item{\textbf{NSIDE}: \textit{integer} Only for HEALPix beams. The NSIDE of the HEALPix map. (nside)}
+\item{\textbf{ORDERING}: \textit{string} Only for HEALPix beams. The ordering parameter of the HEALPix map, allowed values are `nested' and `ring'. (ordering)}
+\end{itemize}
+
+\subsection{Beam Data}
+The axes of the data array in the primary HDU depends on the pixelization scheme.
+
+\subsubsection{HEALPix beam axes}
+For HEALPix beams, the data axes (values of the CTYPE\textit{m} keywords in order) are: 
+\begin{enumerate}
+\item{\textbf{PIX\_IND}: index into the HEALPix pixel array (in the HPX\_INDS extension) This is needed because partial sky maps are supported so pixels may be missing (full sky maps are not handled differently, they have the same required elements as partial maps). The corresponding CRVAL\textit{m}, CRPIX\textit{m}, and CDELT\textit{m} values should all be 1. (pixel\_array)}
```

Comment:
A question because I'm not sure - does the HEALPix definition of these pixels fully constrain the coordinate system? For example, I know there are ways of going from pixel index to "theta" and "phi" angles, but are those directly related to za and az?

Reply:
Healpix is a mapping of the sphere in a generic theta, phi space. Usually theta, phi map to RA, Dec in sky maps, but for beams the sensible thing is for them to map to az, za. This is specified in the description of the `pixel_coordinate_system` in the docs, but I can add some more language to this memo as well.

Reply:
Ah, I was looking at old docs. This and the next question are already addressed.